components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -103,31 +151,38 @@ function createGMIntroMessage(channel: Channel, centeredIntro: string, profiles:
                 <ProfilePicture
                     key={'introprofilepicture' + profile.id}
                     src={Utils.imageURLForUser(profile.id, profile.last_picture_update)}
-                    size='xl'
+                    size='xl-custom-GM'
                     userId={profile.id}
                     username={profile.username}
                 />
             ));
-
         return (
             <div
                 id={channelIntroId}
-                className={'channel-intro ' + centeredIntro}
+                className={'channel-intro d-flex flex-column ' + centeredIntro}
             >
                 <div className='post-profile-img__container channel-intro-img'>
                     {pictures}
                 </div>
-                <p className='channel-intro-text'>
-                    <FormattedMarkdownMessage
+                <h2 className='channel-intro__title'>
+                    {channel.display_name}
+                </h2>
+                <p className='channel-intro__content'>
+                    <FormattedMessage
                         id='intro_messages.GM'
-                        defaultMessage='This is the start of your group message history with {names}.\nMessages and files shared here are not shown to people outside this area.'
+                        defaultMessage='This is the start of your group message history with {names}.\nMessages and files shared here are not shown to anyone else outside the group.'
                         values={{
                             names: channel.display_name,
                         }}
                     />
                 </p>
-                {createBoardsButton(channel, boardComponent)}
-                {createSetHeaderButton(channel)}
+                <div className='d-flex flex-row'>
+                    {createBoardsButton(channel, boardComponent)}
+                    {createAddPeopleButton(channel, isInvitingPeople, createClasses?.(false))}
```

Comment:
I've wondered about this before:
All of these `create....` functions can just be regular components. That way we have way more possibilities, are more performant and are more flexible. Since this is a PR aiming at improving the Intro messages altogether we should look into shifting to components here